doc/cprover-manual/loop-invariant-synthesizer.md

```diff
@@ -0,0 +1,37 @@
+[CPROVER Manual TOC](../)
+
+## Loop Invariant Synthesis with goto-instrument
+
+This is a loop invariant synthesizer built in goto-instrument that 
+synthesize [loop invariants](contracts-invariants.md) for a 
+pre-existing GOTO binary, annotate the invariant clause into 
+the GOTO binary, and produce an annotated binary as output.
+
+All synthesized loop invariants must be inductive, i.e., 
+all [loop-invariant checks](contracts-invariants.md#semantics)
+succeed in the instrumented GOTO binary.
+
+For different synthesis strategies, there are different guarantees 
+beyond the inductiveness on the synthesized loop invariants, 
+summarized in the following table.
+
+Strategy         | Flag                                           | Guarantees
+-----------------|------------------------------------------------|----
+Enumerative      | --synthesize-loop-invariants-enumerative       | All checks will succeeded in the annotated GOTO binary
```

Comment:
What happens if the asserts (I presume that is what you mean by checks) are false?